infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -58,6 +58,23 @@
         </encoder>
     </appender>
 
+    <appender name="log_unit_writes_file" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>${LOG_DIRECTORY}/tx-log.log</file>
```

Comment:
`tx-log.log` and `log-unit.%i.log.gz`? Should they just be the same?

Reply:
fixed